### PR TITLE
Feature/revert security workarounds

### PR DESCRIPTION
**Pull request by Brutus5000**

### CODE REVIEW

**Comment by bukajsytlos** [comment]
lgtm

**Review by bukajsytlos — DISMISSED** [review]
lgtm

File: src/main/java/com/faforever/api/data/domain/MapVersionStatistics.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 @Entity
 @Setter
 @Table(name = "map_version_statistics")
-@Include(rootLevel = true, type = com.faforever.api.dto.MapVersionStatistics.TYPE)
+@Include(rootLevel = true, type = "mapVersionStatistics")
```
Comment (1-alex98) [inline src/main/java/com/faforever/api/data/domain/MapVersionStatistics.java:19]: extract in constant

File: src/main/java/com/faforever/api/data/domain/MapVersionReview.java [hunk 2]
```diff
@@ -15,7 +15,7 @@
 import javax.persistence.Table;
 
 @Setter
-@Include(rootLevel = true, type = com.faforever.api.dto.MapVersionReview.TYPE)
+@Include(rootLevel = true, type = "mapVersionReview")
```
Comment (1-alex98) [inline src/main/java/com/faforever/api/data/domain/MapVersionReview.java:18]: extract in constant

File: src/main/java/com/faforever/api/data/domain/LobbyGroup.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+/**
+ * @deprecated LobbyGroups are supposed to be replaced with role based security
+ */
+@Entity
+@Table(name = "lobby_admin")
+@Setter
+@Deprecated
+@NoArgsConstructor
+@AllArgsConstructor
+@Include(type = "lobbyGroup")
```
Comment (1-alex98) [inline src/main/java/com/faforever/api/data/domain/LobbyGroup.java:24]: extract in constant

File: src/main/java/com/faforever/api/data/domain/Ladder1v1Map.java [hunk 4]
```diff
@@ -24,7 +24,7 @@
 @Entity
 @Setter
 @Table(name = "ladder_map")
-@Include(rootLevel = true, type = com.faforever.api.dto.Ladder1v1Map.TYPE)
+@Include(rootLevel = true, type = "ladder1v1Map")
```
Comment (1-alex98) [inline src/main/java/com/faforever/api/data/domain/Ladder1v1Map.java:27]: extract in constant

File: src/main/java/com/faforever/api/data/domain/GroupPermission.java [hunk 5]
```diff
@@ -29,7 +29,7 @@
  */
 @Entity
 @Table(name = "group_permission")
-@Include(rootLevel = true, type = com.faforever.api.dto.GroupPermission.TYPE)
+@Include(rootLevel = true, type = "groupPermission")
```
Comment (1-alex98) [inline src/main/java/com/faforever/api/data/domain/GroupPermission.java:32]: extract in constant

File: src/main/java/com/faforever/api/data/domain/GlobalRating.java [hunk 6]
```diff
@@ -7,6 +7,6 @@
 
 @Entity
 @Table(name = "global_rating")
-@Include(rootLevel = true, type = com.faforever.api.dto.GlobalRating.TYPE)
+@Include(rootLevel = true, type = "globalRating")
```
Comment (1-alex98) [inline src/main/java/com/faforever/api/data/domain/GlobalRating.java:10]: extract in constant

File: src/main/java/com/faforever/api/data/domain/GameReview.java [hunk 7]
```diff
@@ -15,7 +15,7 @@
 import javax.persistence.Table;
 
 @Setter
-@Include(rootLevel = true, type = com.faforever.api.dto.GameReview.TYPE)
+@Include(rootLevel = true, type = "gameReview")
```
Comment (1-alex98) [inline src/main/java/com/faforever/api/data/domain/GameReview.java:18]: extract in constant

File: src/main/java/com/faforever/api/data/domain/GamePlayerStats.java [hunk 8]
```diff
@@ -18,7 +18,7 @@
 
 @Entity
 @Table(name = "game_player_stats")
-@Include(rootLevel = true, type = com.faforever.api.dto.GamePlayerStats.TYPE)
+@Include(rootLevel = true, type = "gamePlayerStats")
```
Comment (1-alex98) [inline src/main/java/com/faforever/api/data/domain/GamePlayerStats.java:21]: extract in constant

File: src/main/java/com/faforever/api/data/domain/Game.java [hunk 9]
```diff
@@ -31,7 +31,7 @@
 
 @Entity
 @Table(name = "game_stats")
-@Include(rootLevel = true, type = com.faforever.api.dto.Game.TYPE)
+@Include(rootLevel = true, type = "game")
```
Comment (1-alex98) [inline src/main/java/com/faforever/api/data/domain/Game.java:34]: extract in constant

File: src/main/java/com/faforever/api/data/domain/FeaturedMod.java [hunk 10]
```diff
@@ -81,15 +78,8 @@ public String getFileExtension() {
     return fileExtension;
   }
 
-  @Column(name = "deployment_webhook")
+  @Column(name = "deployment_webhook ")
```
Comment (1-alex98) [inline src/main/java/com/faforever/api/data/domain/FeaturedMod.java:81]: is that space intended?

File: src/main/java/com/faforever/api/data/domain/DomainBlacklist.java [hunk 11]
```diff
@@ -19,7 +19,7 @@
 @Entity
 @Setter
 @Table(name = "email_domain_blacklist")
-@Include(rootLevel = true, type = com.faforever.api.dto.DomainBlacklist.TYPE)
+@Include(type = "domainBlacklist", rootLevel = true)
```
Comment (1-alex98) [inline src/main/java/com/faforever/api/data/domain/DomainBlacklist.java:22]: extract in constant

File: src/main/java/com/faforever/api/data/domain/CoopMap.java [hunk 12]
```diff
@@ -19,7 +18,7 @@
 @Entity
 @Table(name = "coop_map")
 @EntityListeners(CoopMapEnricher.class)
-@Include(rootLevel = true, type = CoopMission.TYPE)
+@Include(rootLevel = true, type = "coopMission")
```
Comment (1-alex98) [inline src/main/java/com/faforever/api/data/domain/CoopMap.java:21]: extract in constant

File: src/main/java/com/faforever/api/data/domain/ClanMembership.java [hunk 13]
```diff
@@ -16,7 +16,7 @@
 
 @Entity
 @Table(name = "clan_membership")
-@Include(rootLevel = true, type = com.faforever.api.dto.ClanMembership.TYPE)
+@Include(rootLevel = true, type = "clanMembership")
```
Comment (1-alex98) [inline src/main/java/com/faforever/api/data/domain/ClanMembership.java:19]: extract in constant

File: src/main/java/com/faforever/api/data/domain/BanInfo.java [hunk 14]
```diff
@@ -28,7 +28,7 @@
 
 @Entity
 @Table(name = "ban")
-@Include(rootLevel = true, type = com.faforever.api.dto.BanInfo.TYPE)
+@Include(rootLevel = true, type = "banInfo")
```
Comment (1-alex98) [inline src/main/java/com/faforever/api/data/domain/BanInfo.java:31]: extract in constant

**Review by 1-alex98 — CHANGES_REQUESTED** [review]

**Comment by Brutus5000** [comment]
I closed all remarks from alex as these are revert commits. I will probably fix it if I split out the dtos again.

**Pull request merged**